TestAutomationCourse/src/learningpersonalprocess/Main.java

```diff
@@ -0,0 +1,118 @@
+package learningpersonalprocess;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+//        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+//        System.out.println("Enter contact:");
+//        String contact = reader.readLine();
+//
+//        System.out.println("Getting number for contact " + contact);
+//        String contactNumber = getNumberForContact(contact);
+//        System.out.println("The number of requested contact is [" + contactNumber + "]");
+//
+//        Flower whiteRose = Flower.createAFlower("rose", "white");
+//        whiteRose.setColor("white");
+//
+//        Flower redFlower = Flower.createAFlower("rose", "red");
+//        redFlower.setColor("red");
+//        redFlower.getName();
+//        String name = redFlower.name;
+//
+//
+//        Flower sunflower = Flower.createAFlower("sunFlower","yellow");
+//        sunflower.setColor("yellow");
+//
+//        System.out.println("WhiteRose has color " + whiteRose.getName() + ".\nSunflower is " + sunflower.getName()
+//                + "\nRedFlower is " + redFlower.getName());
+//        System.out.println("whiteFLower[" + whiteRose.nrOfPetals
+//                + "] redFlower[" + redFlower.nrOfPetals
+//                + "] sunFlower[" + Flower.nrOfPetals + "]");
+//
+//        redFlower.nrOfPetals = 50;
+//        System.out.println("WhiteFloer nrOf petals " + whiteRose.nrOfPetals);
+
+
+
+        String name = "123454321";
+        String result = "";
+        for (int i = name.toLowerCase().length() - 1; i >= 0; i--) {
```

Comment:
.toLowerCase() method call isn't relevant here as it's result is ignored, you're simply getting the lenght of the string